### PR TITLE
chore(*): added russian language support

### PR DESCRIPTION
**Pull request by exoriri**
Just added monthes names in russian language

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -3203,6 +3203,8 @@ ansi-red@^0.1.1:
   version "0.1.1"
   resolved "https://registry.yarnpkg.com/ansi-red/-/ansi-red-0.1.1.tgz#8c638f9d1080800a353c9c28c8a81ca4705d946c"
   integrity sha1-jGOPnRCAgAo1PJwoyKgcpHBdlGw=
+  dependencies:
```
Comment (maggialejandro) [inline yarn.lock:3206]: please remove this file changes

**Review by maggialejandro — COMMENTED** [review]
🚀

File: yarn.lock (hunk 1 above)
Comment (exoriri) [reply yarn.lock]: yeah, sure. Done✅

**Comment by maggialejandro** [comment]
Please amend the last commit to the previous one 🙏

**Comment by exoriri** [comment]
> Please amend the last commit to the previous one 🙏

Sorry, Made a lot of commits, but last one is right

**Comment by maggialejandro** [comment]
It should be only one commit, please fix that so I can merge this 🙏

**Comment by exoriri** [comment]
> It should be only one commit, please fix that so I can merge this 🙏

finally, one commit left 🙂

File: yarn.lock [hunk 2]
```diff
@@ -9995,4 +9995,4 @@ yargs@^16.2.0:
     require-directory "^2.1.1"
     string-width "^4.2.0"
     y18n "^5.0.5"
-    yargs-parser "^20.2.2"
```
Comment (maggialejandro) [inline yarn.lock:9998]: remove this change

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-calendario",
-  "version": "3.2.0",
+  "version": "3.2.1",
```
Comment (maggialejandro) [inline package.json:3]: no need to change this

File: yarn.lock (hunk 2 above)
Comment (exoriri) [reply yarn.lock]: done

File: package.json (hunk 3 above)
Comment (exoriri) [reply package.json]: done

**Review by maggialejandro — APPROVED** [review]

**Comment by maggialejandro** [comment]
Thank you! 🚀 @exoriri

**Pull request merged**